docs/docs/mapping/customizing_openapi_output.md

```diff
@@ -284,3 +284,96 @@ Output json:
 },
 ```
 {% endraw %}
+
+### Hiding fields, methods, services and enum values
+
+If you require internal or unreleased fields and APIs to be hidden from your API documentation, [`google.api.VisibilityRule`](https://github.com/googleapis/googleapis/blob/master/google/api/visibility.proto#L89) annotations can be added customize where they are generated. Combind with the option `visibility_restriction_selectors` overlapping rules will appear in the OpenAPI output. 
+
+Fields and APIs without `google.api.VisibilityRule` annotations will always appear in the generated output.
```

Comment:
I like the wording in the original source better: https://github.com/googleapis/googleapis/blob/9916192ab15e3507e41ba2c5165182fec06120d0/google/api/visibility.proto#L66-L67. 
```suggestion
Elements without `google.api.VisibilityRule` annotations will appear as usual in the generated output.
```